cmd/cloud_script_validate.go

```diff
@@ -0,0 +1,58 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"go.k6.io/k6/cloudapi"
+	"go.k6.io/k6/cmd/state"
+	"go.k6.io/k6/lib/consts"
+)
+
+func getCloudScriptValidateCmd(gs *state.GlobalState) *cobra.Command {
+
+	c := &cmdCloudRunTest{
+		gs:            gs,
+		testID:        testIDNotSet,
+		showCloudLogs: true,
+		exitOnRunning: false,
+		uploadOnly:    false,
+	}
```

Comment:
we need the flags used in the cloud run for the validation.
The logic is coupled with that functionality so it's the easier way to get the feature.
Refactoring out the option validation might take a lot of time and mainly not be accepted as a contribution 🤔

Reply:
I don't see the command used anywhere in the code :eyes: As for the flags, we either can add them directly 
```go
	validateSub.Flags().AddFlagSet(optionFlagSet())
	validateSub.Flags().AddFlagSet(runtimeOptionFlagSet(false))
```
or remove them completely. Validating the script with option flags may be confusing as the flag values will not apply if the test is started from UI/by id. 
We should enforce the test script as a single truth source for the options here. This would also mean that env variables, such as `K6_DURATION` and `K6_VUS`, should be ignored as well, but that can be addressed later

Reply:
> do you mean in demo.sh ?
No-no, I mean the `cmdCloudRunTest` object (it is used for flags, sorry for confusion).

> For the flags is rough, the logic that validates the test will expect the flags and try to use them, if they are not present it will panic.
We can consider factoring it out but I'm not sure if it's worth it, the validation logic is doing a bunch of stuff, merging all the configs & setting default values.

Ok, let's keep it as is :+1: Could you please mention that in the feature doc so that we don't forget about it?